src/test/java/org/mockitousage/verification/VerificationWithTimeoutTest.java

```diff
@@ -137,21 +156,32 @@ public void shouldAllowMixingOnlyWithTimeoutAndFail() throws Exception {
      * is used.
      */
     @Test
-    public void canIgnoreInvocationsWithJunit() {
+    public void canIgnoreInvocationsWithJunit() throws InterruptedException {
+        // given
+        DelayedExecution delayedExecution = new DelayedExecution(executor, mock, 0);
+        delayedExecution.recordAsyncCall('0');
+        delayedExecution.recordAsyncCall('1');
+        delayedExecution.recordAsyncCall('2');
+
         // when
-        callAsyncWithDelay(mock, '0', 0, MILLISECONDS);
-        callAsyncWithDelay(mock, '1', 0, MILLISECONDS);
-        callAsyncWithDelay(mock, '2', 20, MILLISECONDS);
+        delayedExecution.allAsyncCallsStarted();
 
         // then
+        verify(mock, timeout(50)).oneArg('0');
```

Comment:
I will keep original intent of that test. However this particular test is at least weird and I have no idea what this test is actually testing